vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/overview/ProjectOverviewPage.java

```diff
@@ -20,8 +20,8 @@
  * @since 1.0.0
  */
 @PageTitle("Project Overview")
-@Route(value = AppRoutes.MAIN, layout = MainLayout.class)
-@RouteAlias(value = AppRoutes.MAIN_ALIAS)
+@Route(value = AppRoutes.PROJECTS, layout = MainLayout.class)
+@RouteAlias(value = AppRoutes.PROJECTS_ALIAS)
```

Comment:
Would it make sense to also have additional aliases similiar to "home" or "main"? 
https://vaadin.com/docs/next/routing/route

Reply:
I understand but would argue that in the context of an application redirecting the user back to the main-page is a sensible operation. 
Without a "home" or "main" alias the knowledge that the ProjectOverviewPage functions as the landing page is implictit rather than explicit. 
However this is more of a NIT than a blocking issue so i'd approve the PR :)